gupb/model/pandas_profiler.py

```diff
@@ -0,0 +1,164 @@
+from time import time
```

Comment:
To nie jest część modelu, więc raczej do wyrzucenia do osobnego pod-pakietu ("profiler"?).